src/components/shared/DlVirtualScroll/useVirtualScroll.ts

```diff
@@ -879,6 +879,10 @@ export function useVirtualScroll({
             create(
                 tag,
                 {
+                    id: 'draggable',
```

Comment:
the dynamicId prop has added to useVirtualScroll

Reply:
@vadim-popa-opentech do we need it ? if it can be generated automatically by the use-virtual-scroll then just let the function itself generate a new uuid and thats it

Reply:
The id identificator from virtual scroll has deleted and has added by class